### DISCUSSION TITLE
Free SQL Editor for Doris?

### DISCUSSION THREAD

**Discussion by ryanhamilton** [apache/doris#36062] [A - General / Q&A]
Hi,

I'm the author of qStudio a Free SQL IDE that tries to support many databases via JDBC:
https://www.timestored.com/qstudio/

I'm considering which database to add support for next and I was attracted to Doris as I saw you had topped the clickhouse benchmark. We have a lot of existing Clickhouse users and our tool is focussed on analysing data so Doris seems like a good fit.

If you would be happy to help with testing please leave a message here or on this issue:
https://github.com/timeseries/qstudio/issues/67

Thanks,
Ryan

**Comment by ixzc** [comment]
we would love to do this

**Comment by ryanhamilton** [comment]
Sorry things got hectic as we reached the frontpage of HN and went viral right after I posted here
https://news.ycombinator.com/item?id=40562210

I'm back with some time to look at this. 

We want to: Check that Apache Doris works well. 
1. All data types work and are shown in the table
2. Numerical/temporal data gets charted appropriate
3. The server tree and autocomplete work

Is there anyone that could commit to:
1. Help us test all data types
2. Work together to write a guide similar to this but for Doris?  https://www.timestored.com/qstudio/database/clickhouse

**Reply by ixzc** [reply]
okay, good job. you can connect me by slack or email xuzhenchao@selectdb.com to have a detail talk.

**Comment by ryanhamilton** [comment]
I can't find a JDBC driver for Doris. Is one available?
